docs/semgrep-appsec-platform/dashboard-beta.md

```diff
@@ -0,0 +1,217 @@
+
+If you choose to turn off recommended priority filters, **all** findings are displayed.
+
+## Production backlog
+
+This pane displays analytics related to findings detected in your **primary or default branch**. This typically means that the finding, usually a security issue, has made it to production environments.
+
+### Key metrics
+
+| Key metrics    | Description |
+| -------------- | ------ |
+| Total new      | New **Open**, **Fixing**, or **Reviewing** findings, including findings that were fixed or ignored within the time period. |
+| Total fixed    | Total **Fixed** findings. |
+| Total ignored  | Total **Ignored** findings. |
+| Total net new  | New **Open**, **Fixing**, or **Reviewing** findings that were **not** fixed or ignored within the specified time period. Lower values are better. |
+
+:::tip
+A low or 0 value for **Total net new** is ideal as it indicates that findings are being triaged or resolved within the time period.
+:::
+### Charts
+
+| Chart | Description |
+| -------  | ------ |
+| Open backlog         | This tracks the total findings from each scan and displays them. Lower values are better. |
+| Backlog activity | Displays the number of new, net new, fixed, and ignored findings. A greater **Fixed** value is better. |
+
+## Secure guardrails
+
+This provides an overview of how secure guardrails in **PR or MR comments** are used in your organization. Other guardrail interfaces, such as the IDE or `pre-commit`, are not counted in this section.
+
+![Secure guardrails pane](/img/dashboard-guardrails.png)
+_**Figure**. Secure guardrails pane. Hover over the charts to view data for that point in time._
+
+### Key metrics
+
+<!-- vale off -->
+| Key metrics    | Description |
+| -------------- | ------ |
+| Findings shown to devs      | Number of findings shown to developers in PR or MR comments (the numerator) against the total findings count (denominator). An upward or stable trend is better. |
+| Findings fixed before backlog      | Number of findings that were fixed before they could be detected in a default branch or production backlog (numerator) against the total findings count in the specified time period. An upward or stable trend is better. |
+
+<!-- vale on -->
+
+### Charts
+
+| Chart | Description |
+| -------  | ------ |
+| Secure guardrails adoption  | Percent of new findings shown to developers over the specified time period. An upward or stable trend is better. |
+| Guardrails activity | This chart displays a breakdown of the status of findings shown to developers; whether they were ignored, fixed, or remained open. A greater **Fixed** value is better. |
+
+## Most findings by project
+
+A table listing projects from most open findings to least, grouped by product or severity. Lower values are better.
+
+:::tip
+It is recommended to prioritize triage and remediation on the top projects listed in this table, especially if the priority filters are enabled.
```

Comment:
```suggestion
It is recommended that triage and remediation be prioritized for the top projects listed in this table, especially if priority filters are enabled.
```

Reply:
not sure about the "be" form of the verb, but yes to correcting my preposition.